tests/cstor-pool/cstor_integration_test.go

```diff
@@ -309,8 +309,8 @@ var _ = Describe("RAIDZ2 SPARSE SPC", func() {
 })
 
 var _ = AfterSuite(func() {
-	spcClient, err := spc.KubeClient().WithKubeConfigPath(kubeConfigPath)
-	Expect(err).To(BeNil())
+	spcClient := spc.NewKubeClient(spc.WithKubeConfigPath(kubeConfigPath))
```

Comment:
What if kubeconfigpath is not set & still we are trying to use it?

Can we have a function `WithKubeConfigPathIf(path string)` that sets the path only if its not empty !!!
Do you think that will be a good approach.

Reply:
We already use 
```
func (k *Kubeclient) getClientsetForPathOrDirect() (*clientset.Clientset, error) {
	if k.kubeConfigPath != "" {
		return k.getClientsetForPath(k.kubeConfigPath)
	}
	k.WithDefaults()
	return k
	return k.getClientset()
}
```
in spc client to check the kubeconfig path.